### ISSUE TITLE
ORDS needs an admin interface

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ox-it/ords#606]
```
At present, administering the ORDS is a hassle. Putting together a simple 
administrative interface would reduce the staff time required to perform basic 
housekeeping and administrative tasks.

The identified required functionality is as follows:

for admins:
 - change the owner of a project
 - make a project a full project
 - mark a project as deleted
 - get ords stats (no. full and trial projects, number of registered users, number of registered users who are members of full projects, number of users by institution)
 - display the audit for projects

for sysadmins:
 - really delete projects marked as deleted
 - move a project to a different db server
 - update the list of admins and sysadmins



```

Original issue reported on code.google.com by `jajwil...@gmail.com` on 22 Jan 2015 at 2:33

**Comment by thestoat** [comment]
When changing the project owner, 
1. It shall be possible for any ORDS user to be made owner (i.e. not simply another project member and not someone who is not a member of ORDS). The new owner shall be notified of the change by email.
2. The old owner shall be made a project administrator and shall also be notified of this change by email.

When a project is made a full project or marked as deleted, the owner shall be notified by email.

When an user is made an administrator (or stops being so), they shall be notified by email.

**Comment by thestoat** [comment]
The following has been done:

for admins:
- change the owner of a project
- make a project a full project
- mark a project as deleted
- get ords stats (no. full and trial projects, number of registered users)
- display the audit for projects

for sysadmins:
- really delete projects marked as deleted (currently disabled since dangerous - will enable shortly)
- update the list of admins and sysadmins (currently sysadmins can view this information only)

**Comment by jajwilson** [comment]
This is taking shape nicely, although I notice that in Dev there are a lot of projects named 'Test Project' or 'Test P1....' with no known owner. I am assuming that these are being generated automatically for system-testing purposes, but should they not be being deleted automatically as well?

**Comment by thestoat** [comment]
Yes, indeed they should. Sometimes a test will fail and not clean after itself. I am trying to chase those down and fix them. Similarly, there are a number of "Test User" entries ... I am currently cleaning these up

**Comment by thestoat** [comment]
Should now be ok.
